### ISSUE TITLE
Cannot activate space_after_anon_function in Vue

### ISSUE THREAD

**Issue by msmarino** [Glavin001/atom-beautify#1327]
# Description

I am trying to activate **space_after_anon_function** in **Vue** files.  I have activated it in Beautify under **CoffeeScript**, **JavaScript**, **EJS** & **JSX**, each with **Jslint happy** on and off (where applicable).

# Expected Results

The beautified code should have looked like:

```
...
export default {
  methods: function () {
    console.log("Hi")
  }
}
...
```

# Steps to Reproduce

1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!

# Debug

Here is a link to the `debug.md` Gist: https://gist.github.com/4dbecf350c4a7b95bb4a5ab5711da5cc

# Checklist

- [x] I have tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] I have reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched for existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue
- [x] Generated debugging information and added link for `debug.md` Gist to this issue

**Comment by msmarino** [comment]
Code should have been
```
export default {
  methods: {
    test: function () {
      console.log("Hi")
    }
  }
}
```
but it makes no difference.  Still can't get it to work.

**Comment by Glavin001** [comment]
I see in your `debug.md` you are only using the `Config Options` (Atom-Beautify's package settings) not the `Project Options` (`.jsbeautifyrc`). I do see `space_after_anon_function` is not available for Vue: https://github.com/Glavin001/atom-beautify/blob/master/docs/options.md#vue
You could simply create a `.jsbeautifyrc` file with `{ "vue": { "space_after_anon_funciton": true } }` since all options are passed directly to the Vue beautifier: https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/vue-beautifier.coffee#L10

**Comment by Glavin001** [comment]
~~I was able to reproduce the issue and also dived into some analysis. It appears Pretty Diff works with `space_after_anon_function` however JS-Beautify does not! See my `debug.md` for example: https://gist.github.com/Glavin001/f2ddbae9dac7b26134f00bf41cdd2405#final-options
And JS-Beautify is used within the Vue beautifier: https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/vue-beautifier.coffee#L29~~

~~So this appears (at first glance) to be a JS-Beautify bug.~~

---

**Update**: I was able to get this working using `jslint_happy: false` and `space_after_anon_function: false` together! See https://gist.github.com/Glavin001/9ffe16aaba44c37c087ad56027df0f8b for example.

**Comment by Glavin001** [comment]
I've added `JavaScript` options to Vue docs: https://github.com/Glavin001/atom-beautify/blob/master/docs/options.md#vue 
Hope that helps!

**Comment by Glavin001** [comment]
Published to v0.29.17
